### ISSUE TITLE
crash of torch.jit.script on torchvision maskrcnn

### ISSUE THREAD

**Issue by jingxu10** [pytorch/pytorch#59064]
## To Reproduce

1. compile pytorch from source
1.1. conda install astunparse numpy ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions future six requests dataclasses
1.2. git clone --recursive https://github.com/pytorch/pytorch
1.3. cd pytorch
1.4. git checkout v1.7.0
1.5. git submodule sync
1.6. git submodule update --init --recursive
1.7. export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}
1.8. python setup.py install
2. run the following code

code:
`import torch`
`import torchvision`
`model = torchvision.models.detection.maskrcnn_resnet50_fpn(pretrained=True)`
`model.eval()`
`model_script = torch.jit.script(model)`

Error msg:
Traceback (most recent call last):                                                                                                                                                                                 
  File "test.py", line 6, in <module>
    model_script = torch.jit.script(model)
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_script.py", line 897, in script
return torch.jit._recursive.create_script_module(                                                                                                                                                              
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_recursive.py", line 352, in create_script_module
    return create_script_module_impl(nn_module, concrete_type, stubs_fn)
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_recursive.py", line 406, in create_script_module_impl
script_module = torch.jit.RecursiveScriptModule._construct(cpp_module, init_fn)                                                                                                                                
File "/opt/conda/lib/python3.8/site-packages/torch/jit/_script.py", line 388, in _construct                                                                                                                        
  init_fn(script_module)
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_recursive.py", line 388, in init_fn
    scripted = create_script_module_impl(orig_value, sub_concrete_type, stubs_fn)
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_recursive.py", line 410, in create_script_module_impl
    create_methods_and_properties_from_stubs(concrete_type, method_stubs, property_stubs)
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_recursive.py", line 304, in create_methods_and_properties_from_stubs
    concrete_type._create_methods_and_properties(property_defs, property_rcbs, method_defs, method_rcbs, method_defaults)
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_recursive.py", line 680, in compile_unbound_method
    create_methods_and_properties_from_stubs(concrete_type, (stub,), ())
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_recursive.py", line 304, in create_methods_and_properties_from_stubs
    concrete_type._create_methods_and_properties(property_defs, property_rcbs, method_defs, method_rcbs, method_defaults)
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_recursive.py", line 661, in try_compile_fn
    return torch.jit.script(fn, _rcb=rcb)
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_script.py", line 939, in script
    fn = torch._C._jit_script_compile(
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_recursive.py", line 661, in try_compile_fn
    return torch.jit.script(fn, _rcb=rcb)
  File "/opt/conda/lib/python3.8/site-packages/torch/jit/_script.py", line 939, in script
    fn = torch._C._jit_script_compile(
RuntimeError:
object has no attribute nms:
  File "/opt/conda/lib/python3.8/site-packages/torchvision/ops/boxes.py", line 42
    """
    _assert_has_ops()
    return torch.ops.torchvision.nms(boxes, scores, iou_threshold)
           ~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
'nms' is being compiled since it was called from 'batched_nms'
  File "/opt/conda/lib/python3.8/site-packages/torchvision/ops/boxes.py", line 88
        offsets = idxs.to(boxes) * (max_coordinate + torch.tensor(1).to(boxes))
        boxes_for_nms = boxes + offsets[:, None]
        keep = nms(boxes_for_nms, scores, iou_threshold)
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
        return keep
'batched_nms' is being compiled since it was called from 'RegionProposalNetwork.filter_proposals'
  File "/opt/conda/lib/python3.8/site-packages/torchvision/models/detection/rpn.py", line 261
            boxes, scores, lvl = boxes[keep], scores[keep], lvl[keep]
            # non-maximum suppression, independently done per level
            keep = box_ops.batched_nms(boxes, scores, lvl, self.nms_thresh)
            ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
            # keep only topk scoring predictions
            keep = keep[:self.post_nms_top_n()]
'RegionProposalNetwork.filter_proposals' is being compiled since it was called from 'RegionProposalNetwork.forward'
  File "/opt/conda/lib/python3.8/site-packages/torchvision/models/detection/rpn.py", line 344
        proposals = self.box_coder.decode(pred_bbox_deltas.detach(), anchors)
        proposals = proposals.view(num_images, -1, 4)
        boxes, scores = self.filter_proposals(proposals, objectness, images.image_sizes, num_anchors_per_level)
        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE

        losses = {}

## Expected behavior
Code runs without error

<!-- A clear and concise description of what you expected to happen. -->

## Environment

Please copy and paste the output from our
[environment collection script](https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py)
(or fill out the checklist below manually).

You can get the script and run it with:
```
wget https://raw.githubusercontent.com/pytorch/pytorch/master/torch/utils/collect_env.py
# For security purposes, please check the contents of collect_env.py before running it.
python collect_env.py
```

 - PyTorch Version (e.g., 1.0):1.9.0a0
 - OS (e.g., Linux):Ubuntu 20.04.2 LTS (x86_64)
 - How you installed PyTorch (`conda`, `pip`, source):source
 - Build command you used (if compiling from source):mentioned at beginning
 - Python version:3.8 (64-bit runtime)
 - CUDA/cuDNN version:N/A
 - GPU models and configuration:N/A
 - Any other relevant information:

## Additional context

<!-- Add any other context about the problem here. -->


cc @gmagogsfm

**Comment by jingxu10** [comment]
@jgong5

**Comment by jgong5** [comment]
@jiayisunx @leslie-fang-intel

**Comment by jingxu10** [comment]
Hi @gmagogsfm , did you compile pytorch from source or use the binary installed from pip? This issue only occurs when pytorch is compiled from source.

**Comment by gmagogsfm** [comment]
> Hi @gmagogsfm , did you compile pytorch from source or use the binary installed from pip? This issue only occurs when pytorch is compiled from source.

Hi @jingxu10, sorry for the delay. I was actually compiling from source. Have you tried latest PyTorch? Is the issue still there?

**Comment by jingxu10** [comment]
Hi @gmagogsfm , this issue had been fixed from ipex 1.8.0. we use stock pytorch wheel file from 1.8.0, so no issues now.

**Comment by gmagogsfm** [comment]
Thanks for confirming
